doc/Interns/JessicaLim8/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+May 6th 2020
```

Comment:
forgot to put in docs. This should be Date AND Time please

Reply:
Time was added to the file